backend/hospital_allocation/routes.py

```
@@ -0,0 +1,123 @@
+    # IMPORTANT: Replace "your-username" with your actual Hugging Face username
+    repo_id = "kesavan2006/chennai-hospital-optimizer-models" 
+
+    # Download each file and get its local path
+    gbm_path = hf_hub_download(repo_id=repo_id, filename="hospital_nextday_lgbm.pkl")
+    clf_path = hf_hub_download(repo_id=repo_id, filename="hospital_suitability_lgbm.pkl")
+    infra_path = hf_hub_download(repo_id=repo_id, filename="chennai_infra_used.csv")
+    
+    # Load the models and data from the downloaded paths
+    # Note: Using pickle since that's what was used to save them
+    with open(gbm_path, "rb") as f:
+        gbm = pickle.load(f)
```

Ruff (0.12.2)

`pickle` and modules that wrap it can be unsafe when used to deserialize untrusted data, possible security issue

(S301)